pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,55 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
```

Comment:
The inputs start with `lib` and `stdenv`, then the others.

`nixos` and `callPackage` are not used.

Reply:
Solved in  https://github.com/NixOS/nixpkgs/pull/186697/commits/5559d73af573b80289cd5b3694820613672675bd